src/components/SummaryTotal/SummaryTotal.tsx

```diff
@@ -1,19 +1,20 @@
 import React from 'react'
-import './SummaryData.scss'
+import './SummaryTotal.scss'
 
 interface IProps {
     disabledCode: boolean,
-    itemsValue: number,
+    itemsValue: any,
```

Comment:
why any? better put number | string, or something like that

Reply:
I put any just in order to get code work properly, now I chahged it to number and it works just fine.